CMakeLists.txt

```diff
@@ -5,6 +5,8 @@ option(BUILD_EXAMPLES "Build examples" OFF)
 
 project(qualisys_cpp_sdk)
 
+
+
```

Comment:
Remove new-lines.

Reply:
Done.

Reply:
Great! And thank you for the finding + fix! :)